apps/festival/festival/src/app/dashboard/analytics/buyer/buyer-analytics.component.ts

```diff
@@ -197,6 +198,12 @@ export class BuyerAnalyticsComponent {
     @Inject(APP) public app: App
   ) { }
 
+  ngAfterViewInit(): void {
+    setTimeout(() => {
+      scrollIntoView(this.header.nativeElement);
+    })
```

Comment:
```suggestion
    });
```

What is default duration if you don't provide one ?